### PR TITLE
Add integ test test app

### PR DESCRIPTION
**Pull request by michhyun1**
**Issue #:**

**Description of changes:**
Add a new, smaller test app for running specific integ tests. Also, add a video Binding integ test that covers unbind and bind video element apis. 

**Testing**

1. Have you successfully run `npm run build:release` locally? yes
2. How did you test these changes? Ran the integ tests locally, and ran the test app locally.
3. Can these changes be tested using the demo application? If yes, which demo application can be used to test it? No, this commit introduces a new test app.
4. Do you add, modify, or delete public API definitions? If yes, has that been reviewed and approved? No
5. Do you change the wire protocol, e.g. the request method? If yes, has that been reviewed and approved? No

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: integration/app/video_test/video_test.ts [hunk 1]
```diff
@@ -0,0 +1,348 @@
...
+      const buttonVideo = document.getElementById('button-camera');
+      buttonVideo.addEventListener('click', _e => {
+        new AsyncScheduler().start(async () => {
+          if (this.toggleButton('button-camera')) {
+            try {
+              // Defaults to default video device for now
+              const videoInputList = await this.audioVideo.listVideoInputDevices();
+              const device = videoInputList && videoInputList[0];
+              await this.audioVideo.chooseVideoInputDevice(device);
+              this.audioVideo.startLocalVideoTile();
```
Comment (ltrung) [inline integration/app/video_test/video_test.ts:120]: Could we remove this button camera and just use a startLocalVideoTile button?

**Review by simmkyu — CHANGES_REQUESTED** [review]

File: integration/js/app/video_test/video_test.ts [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+  AudioVideoObserver,
+  MeetingSessionStatus,
+  VideoTileState,
+  AudioVideoFacade,
+  MeetingSession,
+  Versioning,
+  MeetingSessionStatusCode
+} from '../../../../src/index';
+
+  export class DemoMeetingApp implements AudioVideoObserver {
```
Comment (simmkyu) [inline integration/js/app/video_test/video_test.ts:16]: nit: Remove 2-space indentation from this file

**Review by simmkyu — DISMISSED** [review]

File: integration/js/app/video_test/video_test.ts (hunk 2 above)
Comment (michhyun1) [reply integration/js/app/video_test/video_test.ts]: Done, resubmitted.

**Review by simmkyu — DISMISSED** [review]

File: integration/js/app/video_test/package.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (ltrung) [inline integration/js/app/video_test/package.json:1]: This should be at least under the app folder. I dont think we need to create a new package for every test app.

File: integration/js/app/video_test/package.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    "ts-loader": "^8.0.4",
+    "typescript": "^4.0.0",
+    "webpack": "^4.44.2",
+    "webpack-cli": "^3.3.12"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "amazon-chime-sdk-js": "file:../../../..",
+    "aws-sdk": "^2.802.0",
+    "compression": "^1.7.4"
```
Comment (ltrung) [inline integration/js/app/video_test/package.json:34]: Do we need compression? See third point in this issue https://github.com/aws/amazon-chime-sdk-js/issues/628

File: integration/js/app/video_test/server.js [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+          Attendee: attendee,
+        },
+      }, null, 2));
+    } else if (request.method === 'POST' && requestUrl.pathname === '/end') {
+      // End the meeting. All attendee connections will hang up.
+      await chime.deleteMeeting({
+        MeetingId: meetingTable[requestUrl.query.title].Meeting.MeetingId,
+      }).promise();
+      respond(response, 200, 'application/json', JSON.stringify({}));
+    } else if (request.method === 'GET' && requestUrl.pathname === '/fetch_credentials') {
```
Comment (ltrung) [inline integration/js/app/video_test/server.js:90]: You dont need this. This is for messaging session.

File: integration/js/app/video_test/server.js [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+http.createServer({}, async (request, response) => {
+  log(`${request.method} ${request.url} BEGIN`);
+  try {
+    // Enable HTTP compression
+    compression({})(request, response, () => {});
+    const requestUrl = url.parse(request.url, true);
+    if (request.method === 'GET' && requestUrl.pathname === '/') {
+      // Return the contents of the index page
+      respond(response, 200, 'text/html', indexPage);
+    } else if (process.env.DEBUG && request.method === 'POST' && requestUrl.pathname === '/join') {
```
Comment (ltrung) [inline integration/js/app/video_test/server.js:38]: Can you double check that this work ?

File: integration/js/app/video_test/server.js (hunk 5 above)
Comment (michhyun1) [reply integration/js/app/video_test/server.js:90]: Okay, removing this condtion

File: integration/js/app/video_test/package.json (hunk 3 above)
Comment (michhyun1) [reply integration/js/app/video_test/package.json:1]: Done.

File: integration/js/app/video_test/package.json (hunk 4 above)
Comment (michhyun1) [reply integration/js/app/video_test/package.json:34]: No we dont, I remved the dependency. I'll create a seperate task to switch to using express for this server code

File: integration/js/app/video_test/server.js (hunk 6 above)
Comment (michhyun1) [reply integration/js/app/video_test/server.js:38]: Checked it, and it doesnt work for me. We probably dont need this, so Im removing it.

**Review by ltrung — APPROVED** [review]

**Pull request merged**